### DISCUSSION TITLE
About adaLN-single's improvement

### DISCUSSION THREAD

**Discussion by dragonbra** [PixArt-alpha/PixArt-alpha#56] [Q&A] [answered]
What are the main improvements of adaLN-single compared to adaLN? Is there a significant reduction in parameter count and training cost? Because I understand that the parameter count of DiT-XL is also around 0.6B, and it seems that in Figure 6 of the ablation experiments, the FID of the adaLN row is significantly lower than that of adaLN-single, even though it uses more GPU memory. 

Can it be understood that if the original DiT-XL is trained on the data you generated, it could yield better results?

**Comment by lawrence-cj** [comment, marked as answer]
The original DIT-xl has almost 900M parameters, adaLN will take 200+M parameters there. The original AdaLN will take both time embedding and text_embedding as input, however, here we just have time_embedding, not very much information here. Besides, the GPU memory consumption is larger as shown in Fig.6, 29G ---> 23G. At last, FID may not related to the ability of the model, since the AdaLN-single-generated image quality in Fig.6 is not significantly worse than AdaLN. Overall, we just take AdaLN-single for faster training.

**Reply by lawrence-cj** [reply]
Besides, the FID here is evaluated on SAM. The inception model for FID is trained on ImageNet. I'm not sure if the FID score can show the ability gap between the two models. Therefore, I just looked into plenty of images generated by AdaLN and AdaLN-single with my eyes, and the quality is almost the same. In conclusion, I don't think the original DiT-XL will get better results for sure.

**Reply by CiaoHe** [reply]
DiT-XL only has ~670M params

**Reply by k-zha14** [reply]
yeah, DiT-XL has  675M params, showed in Table.4. However, using adaLN-single sounds reasonable.
